README.md

```diff
@@ -1,6 +1,9 @@
 # Aura.Di
 
 [![PDS Skeleton](https://img.shields.io/badge/pds-skeleton-blue.svg?style=flat-square)](https://github.com/php-pds/skeleton)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/niden/Aura.Di/badges/quality-score.png?b=pds-standard)](https://scrutinizer-ci.com/g/auraphp/Aura.Di/?branch=pds-standard)
```

Comment:
Badges are already in Quality section. https://github.com/auraphp/Aura.Di#quality . But I see the PDS skeleton badge is not there. May be we need to fix the badge in Quality section ?

Reply:
Ah I did not notice that sorry. I will move them around

Reply:
Resolved